tst/EnergyPlus/unit/WindowManager.unit.cc

```diff
@@ -2838,6 +2849,1543 @@ TEST_F(EnergyPlusFixture, WindowManager_SrdLWRTest)
+    state->dataZoneEquip->ZoneEquipConfig(1).InletNode(2) = 2;
+    state->dataZoneEquip->ZoneEquipConfig(1).NumExhaustNodes = 1;
+    state->dataZoneEquip->ZoneEquipConfig(1).ExhaustNode.allocate(1);
+    state->dataZoneEquip->ZoneEquipConfig(1).ExhaustNode(1) = 3;
+    state->dataZoneEquip->ZoneEquipConfig(1).NumReturnNodes = 1;
+    state->dataZoneEquip->ZoneEquipConfig(1).ReturnNode.allocate(1);
+    state->dataZoneEquip->ZoneEquipConfig(1).ReturnNode(1) = 4;
+    state->dataZoneEquip->ZoneEquipConfig(1).FixedReturnFlow.allocate(1);
+
+    state->dataLoopNodes->Node.allocate(4);
+    state->dataHeatBal->SurfTempEffBulkAir.allocate(3);
+    state->dataHeatBalSurf->SurfTempInTmp.allocate(3);
+
+    int surfNum1 = UtilityRoutines::FindItemInList("WALL", state->dataSurface->Surface);
+    int surfNum2 = UtilityRoutines::FindItemInList("FENESTRATIONSURFACE", state->dataSurface->Surface);
+    int surfNum3 = UtilityRoutines::FindItemInList("FLOOR", state->dataSurface->Surface);
+
+    state->dataSurface->Surface(surfNum1).HeatTransSurf = true;
+    state->dataSurface->Surface(surfNum2).HeatTransSurf = true;
+    state->dataSurface->Surface(surfNum3).HeatTransSurf = true;
+    state->dataSurface->Surface(surfNum1).Area = 10.0;
+    state->dataSurface->Surface(surfNum2).Area = 10.0;
+    state->dataSurface->Surface(surfNum3).Area = 10.0;
+    state->dataSurface->Surface(surfNum1).SolarEnclIndex = 1;
+    state->dataSurface->Surface(surfNum2).SolarEnclIndex = 1;
+    state->dataSurface->Surface(surfNum3).SolarEnclIndex = 1;
+    state->dataHeatBalSurf->SurfTempInTmp(surfNum1) = 15.0;
+    state->dataHeatBalSurf->SurfTempInTmp(surfNum2) = 20.0;
+    state->dataHeatBalSurf->SurfTempInTmp(surfNum3) = 25.0;
+    state->dataHeatBal->SurfTempEffBulkAir(surfNum1) = 10.0;
+    state->dataHeatBal->SurfTempEffBulkAir(surfNum2) = 10.0;
+    state->dataHeatBal->SurfTempEffBulkAir(surfNum3) = 10.0;
+
+    state->dataLoopNodes->Node(1).Temp = 20.0;
+    state->dataLoopNodes->Node(2).Temp = 20.0;
+    state->dataLoopNodes->Node(3).Temp = 20.0;
+    state->dataLoopNodes->Node(4).Temp = 20.0;
+    state->dataLoopNodes->Node(1).MassFlowRate = 0.1;
+    state->dataLoopNodes->Node(2).MassFlowRate = 0.1;
+    state->dataLoopNodes->Node(3).MassFlowRate = 0.1;
+    state->dataLoopNodes->Node(4).MassFlowRate = 0.1;
+
+    state->dataHeatBalSurf->SurfHConvInt.allocate(3);
+    state->dataHeatBalSurf->SurfHConvInt(surfNum1) = 0.5;
+    state->dataHeatBalSurf->SurfHConvInt(surfNum2) = 0.5;
+    state->dataHeatBalSurf->SurfHConvInt(surfNum3) = 0.5;
+    state->dataHeatBal->Zone(1).IsControlled = true;
+    state->dataHeatBalFanSys->ZoneAirHumRat.allocate(1);
+    state->dataHeatBalFanSys->ZoneAirHumRat(1) = 0.011;
+    state->dataHeatBalFanSys->ZoneAirHumRatAvg.allocate(1);
+    state->dataHeatBalFanSys->ZoneAirHumRatAvg(1) = state->dataHeatBalFanSys->ZoneAirHumRat(1) = 0.011;
+
+    state->dataHeatBalFanSys->MAT.allocate(1);
```

Comment:
Are all of these necessary just to call updateQdotConvOutRep? Same above.

Reply:
Thanks. I removed the un-used initialization setups.